Jenkins/Task1/Jenkinsfile

```diff
@@ -0,0 +1,60 @@
+pipeline {
+    agent { label 'centos' }
+    environment {
+        DOCKER_IMAGE_NAME = "oleksandrstepanov1/build_pet_clinic"

```

Comment:
you dont need no define this in code. Check how to use parameters block and set defaultValue